enterprise/developerbox/installation/board-recovery.md

```diff
@@ -107,25 +114,21 @@ boot messages:
 ~~~
 /*------------------------------------------*/
 /*  SC2A11 "SynQuacer" series Flash writer  */
+/*                                          */
+/*  Version: cd254ac                        */
+/*  Build: 12/15/17 11:25:45                */
 /*------------------------------------------*/
 
 Command Input >
 ~~~
 
-Update the low-level firmware only when strictly necessary, and it
-is recommended to always "practice" by updating the option ROM whether
-or not it requires contains any changes!
+Once the flasher tool is running we are ready flash the CM3 firmware image:
 
 ~~~
-flash write s-mir-cm3
->> Send option ROM via XMODEM <<
-flash write p-master-cm3
+flash write cm3
```

Comment:
Just to check... cm3 works for both pre-installed and upstream firmwares?

Reply:
Yes, "cm3" works for both pre-installed and upstream firmwares.
Actually flasher is different binary from cm3 firmware.
Flasher is located at 0x70000 in NOR Flash, cm3 firmware is located at 0x100000.